src/domain/metadata/command.ts

```diff
@@ -0,0 +1,20 @@
+import type { ReaderTaskEither } from 'fp-ts/lib/ReaderTaskEither'
+import type { MetadataPort } from './port'
+import type { SerializationError } from '@/shared/error/serialize'
+import type { NetworkError } from '@/shared/error/network'
+
+type DataverseItemId = string
+
+export type LoadMetadataError = SerializationError | NetworkError
```

Comment:
```suggestion
export type RetrieveDataverseItemMetadataError = SerializationError | NetworkError
```

I think it's much more related to the command.